internal/cfcontext/context.go

```diff
@@ -0,0 +1,39 @@
+package cfcontext
+
+import (
+	"context"
+	"errors"
+	"syscall/js"
+)
+
+type runtimeCtxKey struct{}
+type incomingPropertyKey struct{}
+
+func New(ctx context.Context, runtimeCtxObj, incomingPropertyObj js.Value) context.Context {
```

Comment:
The `runtimecontext` package is intended to support not only Cloudflare Workers but also, in the future, platforms such as Deno Deploy and AWS Lambda.
Therefore, I'd like to remove features directly related to Cloudflare from this package.
However, since `workers` currently only supports Cloudflare Workers, these proposed changes are acceptable.